app/src/pages/recipes/create.tsx

```diff
@@ -0,0 +1,175 @@
+import React, {ReactElement, useEffect, useState} from 'react';
+import {AuthorType, CreateRecipeForm, IngredientType, OptionType} from '../../interfaces';
+import {CreateRecipeSchema} from "../../schemas";
+import {ErrorMessage, Field, Form, Formik} from 'formik';
```

Comment:
Haven't directly used this yet but it looks like we have a form component in suitcase that would save this dependency. Example [here](https://github.com/travelperk/frontend/blob/main/apps/tk-checkout/src/pages/Checkout/TravelerDetails/DetailsModal/DetailsModalEdit/DetailsForm.tsx#L76-L277).